src/ensembl/src/shared/accordion/components/AccordionItemPanel.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import DisplayName from '../helpers/DisplayName';
+import { DivAttributes } from '../helpers/types';
+import { Consumer as ItemConsumer, ItemContext } from './ItemContext';
+import defaultStyles from '../css/Accordion.scss';
+import classNames from 'classnames';
+
+type Props = DivAttributes & {
+  extendDefaultStyles: boolean;
+};
+
+const AccordionItemPanel = (props: Props) => {
+  const renderChildren = ({ panelAttributes }: ItemContext): JSX.Element => {
```

Comment:
Is this function leftover of a class method? Because there doesn't seem to be any reason for it to exist as a separate function.

Reply:
I guess it exists as a separate function as the children has to be of type `(container: ItemContext) => ReactNode`

Reply:
Further to our discussion, I have removed the need for a separate function.

Reply:
Have you pushed this change? I am still seeing this function here and in the AccordionItemState component.

Reply:
Ahh! Yes, I forgot. Thanks for spotting it :)